engine/table/build.gradle

```diff
@@ -10,15 +10,6 @@ sourceSets {
     main {
         resources {
```

Comment:
Any chance we want to move these resources to src/main/resources to standardize?